project/Dependencies.scala

```diff
@@ -21,15 +21,15 @@ object Dependencies {
   val cassandraVersionInDocs = "4.0"
 
   // Should be sync with the version of the driver in Pekko Connectors Cassandra
-  val driverVersion = "4.15.0"
-  val driverVersionInDocs = "4.14"
+  val driverVersion = "4.17.0"
+  val driverVersionInDocs = "4.17"
 
   val pekkoConnectorsVersion = "1.0.1"
   val pekkoConnectorsVersionInDocs = "1.0"
   // for example
   val pekkoManagementVersion = "1.0.0"
 
-  val logback = "ch.qos.logback" % "logback-classic" % "1.2.13"
+  val logback = "ch.qos.logback" % "logback-classic" % "1.3.14"
```

Comment:
do we need to upgrade logback?

Reply:
no, just aligned the versions with what Pekko Core is using: https://github.com/apache/incubator-pekko/blob/959c98dba58537ac226971efaef576d0ac0fdbbb/project/Dependencies.scala#L34C13-L34C13

Reply:
right, I'll revert that